### PR TITLE
Add boa to Projects

### PR DESCRIPTION
**Pull request by jasonwilliams**

### CODE REVIEW

File: _data/projects/boa.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+tags:
+  - rust
+  - oss
+  - javascript
+  - language
+  - compiler
+  - interpreter
+upforgrabs:
+  name: good first issue
+  link: https://github.com/jasonwilliams/boa/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue"
```
Comment (shiftkey) [inline _data/projects/boa.yml]:
```suggestion
  link: https://github.com/jasonwilliams/boa/labels/good%20first%20issue
```

File: _data/projects/boa.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+stats:
```
Comment (shiftkey) [inline _data/projects/boa.yml]: No need to include these next few lines - the site will fall back to fetching new stats from the API initially, and after this is merged a new Action run will add in the stats

Comment (jasonwilliams) [reply _data/projects/boa.yml]: removed

**Review by shiftkey — APPROVED** [review]
Thanks @jasonwilliams!

**Pull request merged**